packages/site/tests/site.components.e2e.ts

```diff
@@ -0,0 +1,100 @@
+import { Locator, Page, expect, test } from "@playwright/test";
+import {
+  ListOfGeneratedMobileComponents,
+  ListOfGeneratedWebComponents,
+  //@ts-expect-error - No types for mjs file. that is okay.
+} from "../baseComponentLists.mjs";
+
+const buildUniqueComponentList = () => {
+  const combinedList = [
+    ...ListOfGeneratedMobileComponents,
+    ...ListOfGeneratedWebComponents,
+  ].sort();
+
+  const uniqueSet = new Set(combinedList);
+
+  return Array.from(uniqueSet);
+};
+
+const uniqueList = buildUniqueComponentList();
+
```

Comment:
I'm not the most elegant tester so there's probably better way to do a lot of this but as a first pass this works! It cycles through all the component pages making sure they're rendering properly, then it loads the links in right hand nav and follows them. If they're valid storybook pages, the tests pass!